translator/tocwconfig/sampleConfig/advanced_config_darwin.yaml

```diff
@@ -72,11 +72,11 @@ service:
             processors:
                 - ec2tagger
             receivers:
-                - telegraf_mem
                 - telegraf_netstat
                 - telegraf_swap
                 - telegraf_cpu
                 - telegraf_disk
+                - telegraf_mem
```

Comment:
Most of these didn't need to get re-generated. Makes it a bit harder to tell which configurations actually changed, but it's fine.